src/richie/plugins/glimpse/templates/richie/glimpse/glimpse_person.html

```diff
@@ -0,0 +1,38 @@
+{% load thumbnail %}{% spaceless %}
+<div class="person-glimpse">
+    {% if instance.get_link %}
+        <a href="{{ instance.get_link }}"{% if instance.is_blank_target %} target="_blank" rel=”noopener noreferrer”{% endif %}
+        class="person-glimpse__media" tabindex="-1" aria-hidden="true">
+    {% else %}
+        <div class="person-glimpse__media">
+    {% endif %}
+    <img src="{% thumbnail instance.image 200x200 crop upscale subject_location=instance.image.subject_location %}"
+    srcset="{% thumbnail instance.image 200x200 crop upscale subject_location=instance.image.subject_location %} 200w
+        {% if instance.image.width >= 400 %},{% thumbnail instance.image 400x400 crop upscale subject_location=instance.image.subject_location %} 400w{% endif %}
+        {% if instance.image.width >= 600 %},{% thumbnail instance.image 600x600 crop upscale subject_location=instance.image.subject_location %} 600w{% endif %}"
+    sizes="200px"
+    alt="">
+    {% if instance.get_link %}
+        </a>
+    {% else %}
+        </div>
+    {% endif %}
+
+    <div class="person-glimpse__content">
+        <div class="person-glimpse__wrapper">
+            {% if instance.get_link %}
+                <a href="{{ instance.get_link }}"{% if instance.is_blank_target %} target="_blank" rel=”noopener noreferrer”{% endif %}>
```

Comment:
The double quotes wrapping rel attribute are not the good ones. This mistake exists in three html templates.
```suggestion
                <a href="{{ instance.get_link }}"{% if instance.is_blank_target %} target="_blank" rel="noopener noreferrer"{% endif %}>
```

Reply:
It was introduce a while ago and proliferated in the code base taking advantage of copy/paste :space_invader::mask: